tests/release/pipelines/release_to_github.go

```diff
@@ -34,7 +34,7 @@ const (
 	sampCatalogPathInRepo  = "pipelines/release-to-github/release-to-github.yaml"
 )
 
-var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for release-to-github pipeline", Label("release-pipelines", "release-to-github"), func() {
+var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for release-to-github pipeline", Pending, Label("release-pipelines", "release-to-github"), func() {
```

Comment:
This should not be combined into this PR. It should be separate. Two entirely different things

Reply:
OK. I created a new PR for it.